src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -401,7 +401,7 @@ class TaskForm extends React.Component<Props, State> {
                         </PillSelectorDropdown>
 
                         {shouldShowCompletionRule && (
-                            <FeatureFlag feature="activityFeed.tasks.anyTask">
+                            <>
```

Comment:
Are these still needed?

Reply:
The fragment is needed since there are two feature flip values directly within the `()`

Reply:
Gotcha, sorry missed that